src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -203,16 +220,19 @@ def __call__(self, fileToStage):
                 self.completedFiles[fileToStage['LFN']] = fileToStage
 
                 print "===> Stage Out Successful: %s" % fileToStage
+                self.prepareDashboardReport(fileToStage, None, None, None, 'LOCAL', 0)
                 return fileToStage
             except WMException as ex:
```

Comment:
Can you be more specific in the exception you catch? This is the base exception for all WMAgent exceptions. Should we extend this, this might trip you up.